server/auth_function/lambda_function.py

```diff
@@ -155,9 +164,9 @@ def populate_user_if_necessary(db_connection, event) -> None:
         user_name = event["request"]["userAttributes"]["custom:idp_username"]
 
     raw_query = """INSERT INTO app_fam.fam_user
-        (user_type_code, user_guid, cognito_user_id, user_name,
+        (user_type_code, user_guid, cognito_user_id, user_name, business_guid,
         create_user, create_date, update_user, update_date)
-        VALUES( {user_type_code}, {user_guid}, {cognito_user_id}, {user_name},
+        VALUES( {user_type_code}, {user_guid}, {cognito_user_id}, {user_name}, {business_guid},
```

Comment:
just noticed that, if user not exists, we need to insert new user record with the business_guid

Reply:
Good catch.